mesonbuild/optinterpreter.py

```diff
@@ -94,7 +93,9 @@ def IntegerParser(name, description, kwargs):
                                       kwargs['value'],
                                       kwargs.get('yield', coredata.default_yielding))
 
-@FeatureNew('array type option()', '0.44.0')
+# FIXME: Cannot use FeatureNew while parsing options because we parse it before
+# reading options in project(). See func_project() in interpreter.py
+#@FeatureNew('array type option()', '0.44.0')
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [Sider](https://sider.review) -->